packages/rocketchat-api/server/v1/users.js

```diff
@@ -367,23 +367,3 @@ RocketChat.API.v1.addRoute('users.setPreferences', { authRequired: true }, {
 		return RocketChat.API.v1.success({ user: RocketChat.models.Users.findOneById(this.bodyParams.userId, { fields: preferences }) });
 	}
 });
-
-/**
-	This API returns the logged user roles.
-
-	Method: GET
-	Route: api/v1/user.roles
- */
-RocketChat.API.v1.addRoute('user.roles', { authRequired: true }, {
```

Comment:
Instead of removing this right now, let's deprecate this for three releases and then remove it on the fourth release. Yes I know this endpoint is not working as we would expect, however there might be someone who is using it and as soon as we remove it without warning we will break their system.

So, for deprecation example. If this gets added to v0.64, then it will be removed on v0.67. Because it will be deprecated on v0.64, v0.65, and v0.66, then removed on v0.67.